docs/ingredients/accounts.md

```diff
@@ -0,0 +1,41 @@
+
+Lets start by saying that many types of accounts are required to post a minimum rent balance, thus exempting them from paying rent. This includes executable accounts such as programs and other types of account such as tokens. Basically there is a cost to storing the data, and because your placing some collateral up, rent will not get deducted. If and when these accounts are closed they can also collect any remaining balance left over. In general, Rent fees are charged for storing data on-chain. If an account is not rent exempt then a rent fee is deducted based on their space requirements. If an account has no balance left, it will be purged from memory by the runtime after the transaction. Fees are charged based on optimized calculations taking many variables into account, like space used for your account data (more rent and fee links to details below). For many circumstances you will want to create a rent-exempt account. Any account can be made rent-exempt if its balance is higher than some threshold that depends on the space it’s consuming. This can be calculated by calling a get minimum rent function. In most cases, you can close an account and send the remaining balance to a destination address. When sending transactions, some instructions like 'close' may require an account signer.
+
+Signers/Signatures are used as a way to authorize transactions. Transactions may contain instructions that contain actions that require authorizations, such as transfers and account state changes. Remember, when submitting a transaction we include the instructions, accounts, and signatures as required. The various actions and logic inside of a program will determine which accounts and signatures are required. Programs can use these account signatures to authorize the actions that are executed while processing the instructions passed in from transactions. A typical account signer would typically be something like a wallet or keypair. A new type of signer introduced that is able to sign transactions on behalf of a program is called a PDA (Program Derived Address). This allows a program owned account to sign much in the same way a wallet or keypair would. More soon on PDAs.
```

Comment:
We may want to get rid of this section and just say something like "Transactions are signed by an authorized [account](link) in order to be executed successfully" on the Transactions Ingredient.

Reply:
u want to get rid of the top half too that references rent and fees?

Reply:
Depends on if you agree Rent is in a different section. They're two different concepts, but tightly coupled.